src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -37,414 +36,413 @@
+
+        return Uni.createFrom().item(buildK8sResourceList(environment))
+                .emitOn(Infrastructure.getDefaultExecutor())
+                .map(resourceList -> applyDeleteResourceList(environment, resourceList))
+                .onFailure()
+                .recoverWithItem(
+                        throwable -> OperationResponse.builder().environment(environment)
+                                .sideCarServiceUrl("NA").errorMessage(throwable.getMessage())
+                                .originatedFrom(this.getClass().toString()).status(0).build());
+
+    }
+
+    void createNewEnvironment(Environment environment) {
+        if (!nameSpaceExists(environment))
+            createMpPlusProject(environment);
+        KubernetesList result = buildK8sResourceList(environment);
+        LOG.debug("create environment is in progress");
+        processK8sList(result, environment.getTraceID(), environment.getNameSpace());
+    }
+
+    // TODO: this implementation is mp+ specific, using inheritance
+    // create implementations for different cloud providers,
+    // the implementation is determined by an attribute of application.properties
+    private void createMpPlusProject(Environment environment) {
+        Map<String, String> templateParameters = buildTemplateParameterMap(environment);
+        LOG.debug("creating namespace with parameter {}", templateParameters);
+        createNewTenantNamespace(environment, templateParameters);
+        var eb = EventStructure.builder().uuid(environment.getTraceID().toString());
+        eb.websiteName(environment.getWebsiteName()).environmentName(environment.getName()).state(
+                "namespace created successfully in {} cluster".replace("{}", environment.getNameSpace()));
+        prepareNewTenantNameSpace(environment, templateParameters, eb);
+    }
+
+    private Map<String, String> buildTemplateParameterMap(Environment environment) {
+
+        var tenantName = ConfigProvider.getConfig().getValue("mpp.tenant.name", String.class);
+        var ns = (environment.getNameSpace()).replace(tenantName.concat("--"), "");
+        LOG.debug("Creating namespace with name {}", ns);
+        var appCode = ConfigProvider.getConfig().getValue("mpp.app.code", String.class);
+
+        var devOpsNamingConvention = ConfigProvider.getConfig()
+                .getValue("application.devops.naming.convention", String.class);
+
+        return Map.of("APP_CODE", appCode,
+                "TENANT_NAME", tenantName,
+                "NS_NAME", ns,
+                "DEVOPS_NAMING_CONVENTION", devOpsNamingConvention,
+                "DE_NAMESPACE", deDebugNs);
+    }
+
+    private void createNewTenantNamespace(Environment environment, Map<String, String> templateParameters) {
+        var k8sNSList = ocClient
+                .templates()
+                .load(Operations.class.getResourceAsStream("/openshift/mpp-namespace-template.yaml"))
```

Comment:
could you please add this path to a variable ? (also if possible you can provide an option to read this path from the env variable if you think it's feasible)

`private final String OPENSHIFT_NAMESPACE_PATH = "/openshift/mpp-namespace-template.yaml";`